readthedocs/doc_builder/config.py

```diff
@@ -143,9 +144,15 @@ def load_yaml_config(version):
             'type': 'sphinx',
             'name': version.slug,
         })
+        # Use a top-level rtd.yml if it exists, otherwise search for it
+        possible_rtd_yaml = os.path.join(checkout_path, 'readthedocs.yml')
+        if os.path.exists(possible_rtd_yaml):
```

Comment:
I dont like passing the absolute path of file and directory path with the same parameter. it will be confusing in the future.
It can be something like
```python
if os.path.exists(possible_rtd_yaml):
    kwargs = {'file_path': possible_rtd_yaml, 'only': True}
else:
    kwargs = {'path': checkout_path, 'only': False}
```